<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CValueCardDialog</class>
 <widget class="QDialog" name="CValueCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>储值卡</string>
  </property>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>230</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>退出</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>130</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>备注：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="edtComment"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>41</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>卡类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxCardType">
      <item>
       <property name="text">
        <string>EMID卡</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MIFARE卡</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>登记者：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtRecorder">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>卡片使用状态：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxCardStatus">
      <item>
       <property name="text">
        <string>启用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>挂失</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>卡号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtNumber"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>卡片自编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtSelfNumber"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>卡余额：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtRemainder">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>扣费标准：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxStandard"/>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="chkEnter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>入场必须刷卡</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkLeave">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>出场必须刷卡</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
